### ISSUE TITLE
How can i get root node "viewbox" value

### ISSUE THREAD

**Issue by suzhongzhou1991** [exyte/Macaw#465]
let rootRect:Rect = ((node as! SVGCanvas).layout as! SVGNodeLayout).viewBox

but SVGCanvas is private?  Xcode compiles filed

**Comment by sroik** [comment]
Yeah, it's quite annoying. While waiting for PR to be approved you can [fork](https://github.com/exyte/Macaw/pull/478) from fix.
